+neuron/Neuron.m

```diff
@@ -189,6 +189,9 @@ function set_secondorder(~, value)
 
     end
     methods(Static)
+        function quit()
+            warn("Quit on Neuron is not implemented. Close the Matlab session instead.");
```

Comment:
should this just invoke MATLAB's `quit` instead?

Reply:
Better not I think, that might be somewhat unexpected, if people are just trying to quit neuron nicely (based on python examples), while still wanting to continue doing some matlab plotting of results.
It does need to say warning instead of warn.

Maybe do add a comment here also (not part of the warning message) that we are not calling the quit implemented by neuron itself, because it would
- when running inprocess, close the matlab session
- when running outofprocess, give a clib error that the library has unloaded. Which would be true, but from a user perspective not be seen as an error, and thus could be confusing
and therefor we have chosen to override the quit call with a no-op on the matlab side.

Reply:
I thought the Windows also closed the whole matlab session when you did n.quit(), is that desired behaviour?

Reply:
> I thought the Windows also closed the whole matlab session when you did n.quit(), is that desired behaviour?

@ramcdougal please advise!

Reply:
The options that make sense to me are to either shut down everything or to not allow calling `n.quit()`.

Reply:
> Windows runs without any issues if we use Neuron's `n.quit()`. However, I get an error if I _don't_ use that function. Can we build in a check, so that for windows we use the Neuron internal `n.quit()` and for linux/mac we override it with a no-op?

So this is probably what we want to do then

Reply:
Conclusion:

Let's use neuron's internal `quit()` on all platforms. However, on linux/mac we print a warning (before the clib error occurs) that `quit()` can cause errors on linux/mac platforms. We open a new issue for investigating `quit()` on linux/mac.